ui_framework/components/modal/confirm_modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { KuiModal } from '../modal';
+import { KuiModalFooter } from './modal_footer';
+import { KuiModalHeader } from './modal_header';
+import { KuiModalBody } from './modal_body';
+import { KuiModalBodyText } from './modal_body_text';
```

Comment:
All of these are exposed in `../modal`, why not pull them all from there? Or, pull them all from `../index` since they also exist in there? I don't like the root level index personally, but you're already using it below for the button...